backend/app/src/main/resources/db/migration/V1__Initial_ddl_script.sql

```diff
@@ -43,66 +43,56 @@ CREATE TABLE IF NOT EXISTS control (
-  hds_provider_identifier3              varchar(50),
-  hds_provider_identifier_type1         varchar(50),
-  hds_provider_identifier_type2         varchar(50),
-  hds_provider_identifier_type3         varchar(50),
-  hds_hibc_facility_id                  varchar(50),
-  hds_type                              varchar(50),
-  hds_name                              varchar(100),
-  hds_name_alias                        varchar(100),
-  hds_preferred_name_flag               varchar(1),
-  hds_email                             varchar(50),
-  hds_website                           varchar(50),
-  hds_bus_tel_area_code                 varchar(30),
-  hds_bus_tel_number                    varchar(30),
-  hds_tel_extension                     varchar(30),
-  hds_cell_area_code                    varchar(30),
-  hds_cell_number                       varchar(30),
-  hds_fax_area_code                     varchar(30),
-  hds_fax_number                        varchar(30),
-  hds_service_delivery_type             varchar(255),
-  pcn_clinic_type                       varchar(255),
-  pcn_pci_flag                          varchar(1),
-  hds_hours_of_operation                varchar(255),
-  hds_contact_name                      varchar(255),
-  hds_is_for_profit_flag                varchar(1),
-  source_status                         varchar(50),
-  hds_parent_ipc_id                     varchar(19),
-  bus_ipc_id                            varchar(19),
-  bus_cpn_id                            varchar(19),
-  bus_name                              varchar(100),
-  bus_legal_name                        varchar(255),
-  bus_payee_number                      varchar(255),
-  bus_owner_name                        varchar(255),
-  bus_owner_type                        varchar(255),
-  bus_owner_type_other                  varchar(255),
-  fac_building_name                     varchar(100),
-  facility_hds_details_additional_info  varchar(255),
-  physical_addr1                        varchar(255),
-  physical_addr2                        varchar(255),
-  physical_addr3                        varchar(255),
-  physical_addr4                        varchar(255),
-  physical_city                         varchar(255),
-  physical_province                     varchar(255),
-  physical_pcode                        varchar(255),
-  physical_country                      varchar(255),
-  phys_addr_is_private                  varchar(1),
-  mail_addr1                            varchar(255),
-  mail_addr2                            varchar(255),
-  mail_addr3                            varchar(255),
-  mail_addr4                            varchar(255),
-  mail_city                             varchar(255),
-  mail_bc                               varchar(255),
-  mail_pcode                            varchar(255),
-  mail_country                          varchar(255),
-  mail_addr_is_private                  varchar(1),
+  do_not_load_flag VARCHAR(10),
```

Comment:
are the length of the columns approved by Business?

Reply:
They are provided to us by Business